### ISSUE TITLE
Can't play my own music

### ISSUE THREAD

**Issue by sywy1874** [WilliamLCobb/iNDS#223]
When I use the iNDS emulator, I can't play any games and run my music at the same time. Is it possible right now, or is it not?

**Comment by noah978** [comment]
You are correct. As far as I have tested it, it is not currently possible to do that

**Comment by SamuB162** [comment]
I think you can toggle Disable Sound in settings and it should be fine.
